### DISCUSSION TITLE
Harbor Reg. Probleme

### DISCUSSION THREAD

**Discussion by renepetermann289** [PHT-Medic/central#1098] [General]
Hallo zusammen, ich habe ein Problem mit dem bauen des Trains. Ich bekomme wenn ich den Train über Check teste die Fehlermeldung: 

> _builder/check An error with error code registryNotFound occurred_ 

und wenn ich das ganze über Starten mache sagt er mir folgendes: 

> _(HTTP code 500) server error - unauthorized: unauthorized to access repository: master/python/tensorflow, action: pull: unauthorized to access repository: master/python/tensorflow, action: pull_

Folgendes habe ich in den Einstellungen der Registry eingetragen: 

<img width="1320" alt="Pasted Graphic 1 copy" src="https://github.com/PHT-Medic/central/assets/137161550/0ea94d5e-3be8-44bb-8732-4e9a4225ff55">
<img width="1345" alt="Pasted Graffphic" src="https://github.com/PHT-Medic/central/assets/137161550/17f689f1-45a7-4f0d-ba47-1432361320a0">

In Harbor sind die dazugehörigen Projekte ebenfalls eingerichtet.

<img width="851" alt="Pasted Graphic 2" src="https://github.com/PHT-Medic/central/assets/137161550/1584391f-b4f7-4da7-90cd-294366be30fa">

Wird beim bauen des Train der Robot-Accout für das Pullen benutzt? Ich stehe da etwas auf dem Schlauch

**Comment by tada5hi** [comment]
dir fehlen die master images in der harbor registry.

**Reply by renepetermann289** [reply]
Super, ich danke dir für die Antwort. Kann ich die Images bauen und einfach mit push in die harbor registry laden?
